templates/common/_base/units/machine-config-daemon-firstboot.service

```diff
@@ -10,14 +10,16 @@ contents: |
   # We only want to run on 4.3 clusters and above; this came from
   # https://github.com/coreos/coreos-assembler/pull/768
   ConditionPathExists=/sysroot/.coreos-aleph-version.json
-  After=ignition-firstboot-complete.service
+  After=machine-config-daemon-pull.service
   Before=crio.service crio-wipe.service
   Before=kubelet.service
 
   [Service]
   # Need oneshot to delay kubelet
   Type=oneshot
-  ExecStart=/usr/libexec/machine-config-daemon firstboot-complete-machineconfig
+  # Create /usr/local/bin symbolic link unless it exists
+  ExecStart=-/usr/bin/ln --symbolic /usr/libexec/bin/machine-config-daemon /usr/local/bin/machine-config-daemon
```

Comment:
Just curious, why did we add `-` prefix to /usr/bin/ln/ ?

Reply:
This would make systemd ignore the error if the file already exists, https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=